SU-SOE/stanford_soe_helper
# 7.x 2.x soe 3051

# READY FOR REVIEW

# Summary
Fixing Stanford SoE Helper collection feature override.

# Needed By (Date)
- Sprint end

# Criticality
- Getting this into next release


# Steps to Test
- Switch to this branch  -  7.x-2.x-SOE-3051 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-3051)
- Need the library on dev at default/libraries/history.js (https://github.com/browserstate/history.js)
- Need the patched version of views_load_more on dev: `/var/www/ds_jse-soe-dev/public_html/sites/default/modules/contrib/views_load_more`
- Need the patched version of views_ajax_history on dev: `/var/www/ds_jse-soe-dev/public_html/sites/default/modules/contrib/views_ajax_history`
- enable those two contrib modules.
- `drush cc all`
- go to: `/admin/structure/features/stanford_soe_helper_collection` and make sure the feature is not overridden.
- `drush fr stanford_people_spotlight_views`
- go to: `/spotlight` make sure "Giving" is not an option for the "All people" select field and make sure the no results are working as stated in the Related JIRA ticket(s)

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3051

## Related PRs

## More Information

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
